sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1595 @@
+# Release notes Jul'24
+
+Release notes for commit range
+[d2817d6d317db1](https://github.com/intel/llvm/commit/d2817d6d317db1143bb227168e85c409d5ab7c82)
+...
+[ebb3b4a21b3b0e](https://github.com/intel/llvm/commit/ebb3b4a21b3b0e977f44434781729df7de83e436)
+
+## TODO
+
+commit https://github.com/intel/llvm/commit/9876e19f4ff387b35b0c98c7d62e5f50e6de187d
```

Comment:
@tovinkere, @KseniyaTikhomirova, could you please help me understand if this commit should be added into release notes and if so, which message should be used?

Reply:
Talked offline with Kseniya, the commit was dropped from release notes in 82e564153684b5597ce075032cbbc8b2218a2d3f